tests/test_io/test_web/test_load.py

```diff
@@ -66,6 +72,30 @@ def test_biomodels_access(biomodels: Mock) -> None:
     biomodels.get_sbml.assert_called_once_with(model_id="BIOMD0000000633")
 
 
+def test_biomodels_access(biomodels: Mock) -> None:
+    """Test that SBML would be retrieved from the BioModels repository.
+
+    Parameters
+    ----------
+    biomodels : unittest.mock.Mock
+        The mocked object for BioModels model respository.
+
+    """
+    load_model("BIOMD0000000633", cache=False, repositories=[biomodels])
+    biomodels.get_sbml.assert_called_once_with(model_id="BIOMD0000000633")
+
+def test_biomodels_access(embl_gems: Mock) -> None:
```

Comment:
Need a different name here:
```suggestion
def test_embl_gems_access(embl_gems: Mock) -> None:
```